packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -127,6 +131,7 @@ export class ContextSelector extends React.Component<ContextSelectorProps, { oui
             <ContextSelectorContext.Provider value={{ onSelect }}>
               <ContextSelectorMenuList isOpen={isOpen}>{children}</ContextSelectorMenuList>
             </ContextSelectorContext.Provider>
+            {footer && footer}
```

Comment:
Why wouldn't we add `<ContextSelectorFooter>` for the user if they set `footer`, then pass the value of `footer` to `<ContextSelectorFooter>`?

Reply:
We could make the `ContextSelectorFooter` ourselves, we would just need to another another prop to pass any extra props to the Footer element. Should I update it?

Reply:
Ah, thanks. In that case I'll leave it to your judgment. I anticipate the footer will be pretty simple. But if we add a left-aligned variation for the footer, it would be better to pass that directly to the footer, instead of `isFooterLeftAligned` on `<ContextSelector>`.

Reply:
Change to `{footer}`.